spec.md

```diff
@@ -814,7 +815,7 @@ The request provides these query string parameters as useful hints for brokers.
 ##### cURL
 ```
 $ curl 'http://username:password@broker-url/v2/service_instances/:instance_id?accepts_incomplete=true
-  &service_id=service-id-here&plan_id=plan-id-here' -X DELETE -H "X-Broker-API-Version: 2.11"
+  &service_id=service-id-here&plan_id=plan-id-here' -X DELETE -H "X-Broker-API-Version: 2.12"
```

Comment:
I believe there are two missing cURL sections in this version of the diff.  I count 5 and there should be 7